dlt/sources/helpers/rest_client/auth.py

```diff
@@ -30,9 +30,7 @@
 else:
```

Comment:
please revert the changes on the rest helpers, this is under active development by @burnash and will just confuse i think :)

Reply:
Reverted!